style/home-page.css

```diff
@@ -0,0 +1,770 @@
+    flex-direction: row;
+    -ms-flex-pack: distribute;
+    justify-content: space-around;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    height: 13vh;
+  }
+
+  #bottom-section img {
+    height: 200px;
+    width: 200px;
+  }
+
+  #bottom-section ul {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -ms-flex-pack: distribute;
+    justify-content: space-around;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    background-color: transparent;
+    width: 50%;
+    font-size: 16px;
+  }
+
+  .nav-item {
+    color: #525252;
+  }
+
+  .homelink {
+    text-decoration: none;
+    color: #ec5242;
+    border: 4px solid #ec5242;
+    padding: 10px;
+  }
+
+  #intro {
+    height: 100vh;
+  }
+
+  .intro-con {
+    background: url("./images/woodcut.svg") right 206px top 200px no-repeat;
+    background: url("./images/barrel-icon.svg") left 747px bottom 180px no-repeat;
+    background: url("./images/tap.svg") left 60px bottom no-repeat;
+    background: url("./images/b-icon.svg") right -10px no-repeat;
+    background: url("./images/f-leaf.svg") left 60px bottom 350px no-repeat;
+    background: url("./images/f-leaf.svg") right 56px top 430px no-repeat;
+    background-size: 100px 100px;
+    margin-top: -50px;
+    padding-left: 195px;
+  }
+
+  .label {
+    background-size: cover;
+    background-position-y: inherit;
+    background-position-x: inherit;
+  }
+
+  .label h1 {
+    font-size: 65px;
+    line-height: 60px;
+  }
+
+  .welcome-message {
+    font-size: 35px;
+  }
+
+  .description {
+    background-color: #eaeaea;
+    font-size: 15px;
+    width: 50%;
+    padding: 20px 50px;
+    line-height: 25px;
+  }
+
+  .details h2 {
+    font-size: 30px;
+    margin-bottom: 7px;
+    margin-top: 10px;
+  }
+
+  .catering-con {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: horizontal;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+  }
+
+  #catering {
+    height: 60vh;
+    padding: 25px 50px;
+  }
+
+  #catering h2 {
+    color: white;
+  }
+
+  #catering img {
+    width: 100px;
+    height: 100px;
+    margin-bottom: 20px;
+  }
+
+  .beer-selection-con {
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    text-align: center;
+    padding: 10px;
+    border-right: transparent;
```

Comment:
- [ ] The CSS in this view is having some issues , please fix in both pages the home page and about page

![jdzj](https://user-images.githubusercontent.com/61562626/132833500-e4a34b4f-882a-4638-bb3c-340a7aab5033.jpeg)